skstore/ts/examples/sum.ts

```diff
@@ -33,7 +34,11 @@ class T2SIdentify<K extends TJSON, V extends TJSON>
 }
 
 class Add implements Mapper<number, number, number, number> {
-  constructor(private other: EHandle<number, number>) {}
+  private other: EHandle<number, number>;
+
+  constructor(eparams: EHandle<any, any>[], lparams: LHandle<any, any>[]) {
+    this.other = eparams[0];
+  }
```

Comment:
Here is an example of how this approach gives very weak typing: the constructors must uniformly accept an array of `EHandles` and an array of `LHandles` (with no knowledge of their key and value types) and initialize the class fields by picking out the appropriate array elements. This has to be gotten right at map call sites manually. This isn't great at all.